package.json

```diff
@@ -2,13 +2,18 @@
   "name": "auto-wcag",
   "version": "1.0.0",
   "description": "auto-wcag",
-  "main": "index.js",
+	"main": "index.js",
+	"site-url-prefix": "https://auto-wcag.github.io/auto-wcag",
 	"testcases-embeds-dir": "_testcases-embeds/",
-	"testcases-export-dir": "_testcases-export/",
+	"testcases-export-dir": "auto-wcag-testcases/",
+	"testcases-export-filename": "testcases.json",
+	"rules-export-filename": "rules.json",
```

Comment:
Could all these configuration variables be put in a `config` object, which is the standard approach to npm package configuration? https://docs.npmjs.com/files/package.json#config